ietf-te-types.yang

```diff
@@ -1918,7 +1925,8 @@ module ietf-te-types {
        other state or command.";
     reference
       "ITU-T G.808.1 v4.0 (05/2014): Generic protection switching - 
-      Linear trail and subnetwork protection";
+                                     Linear trail and subnetwork
+                                     protection";
```

Comment:
The indentation looks a bit weird ...

Reply:
This is indented per to be aligned with "ITU-T G.808.1 v4.0 (05/2014):"

Please discard the change if this hurts your eyes.